### PR TITLE
Alpaca model, Google Custom Search API integration

### PR DESCRIPTION
**Pull request by c00lerxo**
- add Alpaca model with migration and tests
- add ApiHelper
- add Google Custom Search API Key and Engine ID to environments configs
- run rails rspec:install, as it wasn't run when it was configurated (it was required to create tests)

* I think we should consider adding HTTParty gem and refactor ApiHelper module, as Ruby's HTTP library is obsolete and hard to use.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -323,4 +323,4 @@ RUBY VERSION
    ruby 2.6.1p33
 
 BUNDLED WITH
-   2.0.1
+   1.17.2
```
Comment (voycik) [inline Gemfile.lock:326]: You can update your bundler to v. 2.0.1

**Review by voycik — CHANGES_REQUESTED** [review]

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -33,6 +44,17 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
+<<<<<<< HEAD
```
Comment (katguz3485) [inline spec/rails_helper.rb]: should be removed

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
```
Comment (katguz3485) [inline spec/spec_helper.rb]: It is because both of us used command rspec install ?

**Comment by c00lerxo** [comment]
I broke this pull request ;(

**Review by voycik — APPROVED** [review]

**Comment by c00lerxo** [comment]
Yeaah!!

**Pull request merged**